MetaMask/metamask-extension
# Fix #21179 - Change Optimism chain name to OP Mainnet

## **Description**
Changes "Optimism" chain name to "OP Mainnet" as requested.

## **Manual testing steps**

1.  Open the Network Picker, click the "Add Network" button
2. On the Frequent RPC list, see "OP Mainnet"
3. Add that network, see "OP Mainnet" in the subsequent modals
4. Open the Network Picker, see the "OP Mainnet" item

## **Screenshots/Recordings**


https://github.com/MetaMask/metamask-extension/assets/46655/83faabeb-1dc2-4a80-a0da-9b218c2f2b1f



## **Related issues**

_Fixes #21179

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
